README.md

```diff
@@ -64,7 +65,7 @@ def train(
     epochs: int):
 
     # DDP
-    dist_init(rank, world_size)
+    dist.init_process_group(backend='nccl', init_method="tcp://localhost:29501", rank=rank, world_size=world_size)
```

Comment:
If you don't mind adding a comment stating that this setting is an example (port number for instance?)

Reply:
Never mind, this is just an example, should be fine. Merging as is, thanks again